stdlib/messaging/rabbitmq/src/test/resources/testng.xml

```diff
@@ -22,8 +22,9 @@ under the License.
 <suite name="ballerina-test-suite">
     <!-- Ballerina language Test Cases. -->
     <test name="ballerina-lang-test-suite" parallel="false">
```

Comment:
Shall we rename the test name to `rabbitmq-tests` or something similar?

Reply:
Fixed